shared/defaults.yaml

```diff
@@ -2,13 +2,14 @@
 # -----    Model    -----
 # -----------------------
 model:
-  inchannels: 11
+  inchannels: 10
```

Comment:
Why do you need both this and "channels_to_inc"? Will you ever have the number of in channels not equal to the length of channels_to_inc?

Reply:
You can have channels_to_inc = [0, 1] and inchannels = 4 (due to borders and snow index, ..). 
But yes, inchannels can to be calculated from the combination of other options. Will make this change.